core/arch/riscv/kernel/spinlock.S

```diff
@@ -33,7 +33,7 @@ FUNC __cpu_spin_unlock , :
 END_FUNC __cpu_spin_unlock
 
 /* unsigned int __cpu_spin_trylock(unsigned int *lock) */
-LOCAL_FUNC __cpu_spin_trylock , :
+FUNC __cpu_spin_trylock , :
```

Comment:
For commit "riscv: kernel: spinlock.S: make __cpu_spin_trylock visible":

`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`